client/extensions/woocommerce/app/products/product-create.js

```diff
@@ -190,6 +188,10 @@ class ProductCreate extends React.Component {
 		const isBusy = Boolean( actionList ); // If there's an action list present, we're trying to save.
 		const saveEnabled = isValid && ! isBusy && 0 === this.state.isUploading.length;
 
+		if ( ! product || isNumber( product.id ) ) {
```

Comment:
I'm groking the check for `! product` but the second clause has me thinking a bit - so this is to short circuit the `create` view if a `product` exists and is not a placeholder/un-saved product?

Reply:
Yeah.

I found in testing, the few actions that clear away edits in product UI state, variations, etc (which also resets `currentlyEditingId` and this `product` object) might not have all ran yet before the component mounts. And because of how the CompactTinyMCE component works, (it only an initial value), this is needed to prevent rendering of the form in that brief moment of time. Once the prop updates the form renders.